README.md

```diff
@@ -18,7 +18,7 @@ developed by ACM Hack’s Dev Team: Timothy Gu
 [(@TimothyGu)](https://github.com/TimothyGu/), Jamie Liu
 [(@jamieliu386)](https://github.com/jamieliu386/), Timothy Rediehs
 [(@Timthetic)](https://github.com/Timthetic), Jakob Reinwald, Chandra Suresh,
-Christina Tong, Miles Wu [(@milesswu)](https://github.com/milesswu), and Eric
+Christina Tong[(@christinatong01)](https://github.com/christinatong01), Miles Wu [(@milesswu)](https://github.com/milesswu), and Eric
```

Comment:
It looks like a space is missing between your name and `[`:

![image](https://user-images.githubusercontent.com/1538624/104114964-36c18d80-52d8-11eb-9fad-eb126da4037c.png)